test/e2e/testdata/apisix-gw-config.yaml

```diff
@@ -27,6 +27,7 @@ apisix:
     - 0.0.0.0/0
   port_admin: 9180
   stream_proxy:                 # TCP/UDP proxy
+    only: false
```

Comment:
Please don't forget to update helm chart.

Reply:
done，https://github.com/apache/apisix-helm-chart/pull/127